lib/src/components/NxToggle/NxToggle.scss

```diff
@@ -7,12 +7,17 @@
 @import '../../scss-shared/nx-text-helpers';
 
 // In safari, the width: 0 and -moz-appearance settings fail to hide the <input>.
-// So instead, we use margin on the checkbox and corresponding negative margin on the toggle to hide
+// So instead, we use negative margin on the checkbox and offsetting positive margin on the toggle to hide
 // the <input> under the toggle
-$nx-toggle-input-hide-hack-spacing: 5px;
+$nx-toggle-input-hide-hack-spacing: var(--nx-spacing-2x);
+
+$nx-toggle-control-width: 44px;
 
 .nx-toggle {
-  display: flex;
+  column-gap: 16px;
+  display: grid;
+  grid-template-areas: "toggle content";
+  grid-template-columns: $nx-toggle-control-width 1fr;
```

Comment:
I think the first column can just be `auto`

Reply:
> I think the first column can just be `auto`

It can but:
1. why does it matter?
2. Why be _less_ precise?

Reply:
> 1. why does it matter?

It minimizes use of magic numbers, and in this case removes the need for the `$nx-toggle-control-width` variable.

> 2. Why be less precise?

IMO it's more precise to say "make it the width of its contents" than to say "make it 44px wide... which btw should be the width of its contents".

Reply:
> > 1. why does it matter?
> 
> It minimizes use of magic numbers, and in this case removes the need for the `$nx-toggle-control-width` variable.
> 

Well it's not like the magic number is going away, it's still required.

> > 1. Why be less precise?
> 
> IMO it's more precise to say "make it the width of its contents" than to say "make it 44px wide... which btw should be the width of its contents".

Meh, I think it's a stretch to say that `auto` is _more_ precise than `44px`.

It also doesn't matter to me as long as it works: https://github.com/sonatype/sonatype-react-shared-components/pull/410/commits/04eced974483834ca09e87ee06c09e45aa69196b